gen_cert.py

```diff
@@ -1841,8 +1834,32 @@ def fontlist_with_style(a_style):
         yOffset = minYOffset + ((max_height - height) / 2)
         paragraph.drawOn(PAGE, GUTTER_WIDTH - (name_style.fontSize / 12), yOffset)
 
+
+        # Parse Translations:
+        translated_success_text = ""
+        translated_interstitial_intro = ""
+        translated_disclaimer = ""
+        translated_verify_text = ""
+
+        if self.default_translations and self.locale in self.default_translations:
+            translated_success_text = self.default_translations[self.locale]['success_text']
+            translated_interstitial_intro = self.default_translations[self.locale]['interstitial_intro']
+            translated_disclaimer = self.default_translations[self.locale]['disclaimer']
```

Comment:
Can we just keep calling this `disclaimer_text` instead of renaming the variable?

We generally just assume that text is/can be translated, without explicitly marking it as such..